From 936da10d17a5596b8404bbf79e4a94162182936a Mon Sep 17 00:00:00 2001
From: Valentin Churavy <v.churavy@gmail.com>
Date: Tue, 9 Feb 2021 19:42:22 -0500
Subject: [PATCH 3/3] job-manager: Use correct format string for size_t

Problem: Compilation for 32bit fails due to a format string that assumes
`long int` instead of `size_t`.

Use `zu` instead of `ld`.
---
 src/modules/job-manager/prioritize.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/job-manager/prioritize.c b/src/modules/job-manager/prioritize.c
index 32299f9c4..e1efe4249 100644
--- a/src/modules/job-manager/prioritize.c
+++ b/src/modules/job-manager/prioritize.c
@@ -236,7 +236,7 @@ int reprioritize_all (struct job_manager *ctx)
     /*  Update scheduler with any changed priorities */
     if (sched_prioritize (ctx->h, priorities) < 0) {
         flux_log_error (ctx->h,
-                        "reprioritize: sched.priority: failed for %ld jobs",
+                        "reprioritize: sched.priority: failed for %zu jobs",
                         json_array_size (priorities));
         goto error;
     }
-- 
2.30.0

